src/chat.cpp

```diff
@@ -1131,6 +1225,152 @@ CCMD( say_team )
 	}
 }
 
+//*****************************************************************************
+//
+// [AK] Allows players (or the server) to send private messages to other players.
+//
+bool chat_CanSendPrivateMessage( ULONG ulPlayer )
```

Comment:
The name of the function sounds as if it would only check whether `ulPlayer` is allowed to send a private message, but what it actually does it to change the global variable `g_ulChatPlayer` to a valid private chat target. The name of the function should reflect that. I would also rename the argument `ulPlayer` to `ulSender`.